itests/kit/ensemble.go

```diff
@@ -194,7 +194,7 @@ func (n *Ensemble) FullNode(full *TestFullNode, opts ...NodeOpt) *Ensemble {
 		require.NoError(n.t, err)
 	}
 
-	key, err := key.GenerateKey(types.KTBLS)
+	k, err := key.GenerateKey(types.KTBLS)
```

Comment:
Any reason for the var name change?

Reply:
linter doesn't like shadowed names. `key` is the name of a package, so we should use something else for the variable.